### ISSUE TITLE
How to use main branch package in app

### ISSUE THREAD

**Issue by phlegx** [bootstrap-vue-next/bootstrap-vue-next#1194]
**What you are trying to do.**

I want to use Github main branch bootstrap-vue-next in my local app, to be able to continue development with unreleased code of bootstrap-vue-next.

Is there a way to do that? If yes, can it be added to the docs?

**What you have already tried.**

To add bootstrap-vue-next Github main branch to package.json.

**Any error messages you are getting.**

`error Can't add undefined: invalid package version undefined.`

**Environment (please complete the following information):**

- OS: Linux
- Package Version: main branch
- Npm/Yarn/Pnpm Version: yarn@1.22.19

**Comment by VividLemon** [comment]
This is more of a question about what you can do with npm. 

You are able to reference exact github repositories in package.json, it will pull it an exact copy of the github branch.

However, this comes with the drawback of that simply pulling the github repository doesn't contain a dist folder. Trying to resolve this can have many issues. You could build your own dist, but the exact structure is messed up, as the project is a monorepo and doesn't have "exports" in the root package.json.

You could manually modify these values in the fetched package.json, but that it's a perfect solution as subsequent `npm i` runs will overwrite these values. 

tldr is that you "might be able to", but ultimately it's a bit of a hassle. Feel free to ask follow up questions, do some research and post it here, etc, but I am going to close this as not planned.
